Website/src/rendering/src/components/Forms/AttendeeForm.tsx

```diff
@@ -11,6 +16,28 @@ const AttendeeForm = (): JSX.Element => {
   const [lastName, setLastName] = useState('');
   const [email, setEmail] = useState('');
 
+  useEffect(() => {
+    const getUserData = async () => {
```

Comment:
Would it be possible to extract this function into a helper file, given that it is used in all three forms?